x/ecocredit/client/util.go

```diff
@@ -31,6 +31,15 @@ func mkQueryClient(cmd *cobra.Command) (ecocredit.QueryClient, sdkclient.Context
 	return ecocredit.NewQueryClient(ctx), ctx, err
 }
 
+func WriteMsgCreateBatchJSON(clientCtx sdkclient.Context, batchFile string, msg *ecocredit.MsgCreateBatch) error {
```

Comment:
This function is only used in `testsuite/tx.go`. Can we move this to `testsuite/tx.go`?